modules/data.remote/inst/remote_process.py

```diff
@@ -41,17 +42,19 @@ def remote_process(
     
     end (str) -- ending date area of the data request in the form YYYY-MM-DD
 
-    source (str) -- source from where data is to be downloaded, e.g. "gee" or "appEEARS" etc. Currently only "gee" implemented
+    source (str) -- source from where data is to be downloaded, e.g. "gee" or "appeears" 
 
-    collection (str) -- dataset or product name as it is provided on the source, e.g.  "LANDSAT/LC08/C01/T1_SR",  "COPERNICUS/S2_SR" for gee
+    collection (str) -- dataset or product name as it is provided on the source, e.g. "COPERNICUS/S2_SR" for gee or "SPL3SMP_E.003" for appeears
 
     scale (int) -- pixel resolution, None by default, recommended to use 10 for Sentinel 2
 
+    projection (str) -- type of projection. Only required for appeears polygon AOI type. None by default. 
+
     qc (float) -- quality control parameter, only required for gee queries, None by default
 
     algorithm (str) -- algorithm used for processing data in process_data(), currently only SNAP is implemented to estimate LAI from Sentinel-2 bands, None by default
 
-    output (dict) -- "get_data" - the type of output variable requested from get_data module, "process_data" - the type of output variable requested from process_data module
+    process_data (str) -- the type of output variable requested from process_data module
```

Comment:
I have also removed the `output` dict and replaced it with `process_data` because the `get_data` variable wasn't being used anywhere other than in `process_remote_data` and I felt that it was a bit confusing.

Reply:
I see what you mean, since we changed the logic for gee function names to use collection names instead :+1: 

But I was wondering if we are losing some generality this way, now you hardcode `bands` in the `process_remote_data`, it's true that we only have one processing function (`bands2lai_snap`) for now, but it may not be the case in the future (just making this up, `images2ndvi_algX`). I can't think of an example for now, but for some future remote sources we might also need the original logic in case we need to formulate functions like `source2pecan_output$get_data`. So I would vote for keeping it.

Reply:
That's right. Ok, then what should be the value of `get_data` say in case of AppEEARS?

Reply:
Could the default values of this list for both levels be `None`? Since the current implementation of AppEEARS also doesn't use `process_data` Or it can always stay as `bands` by default.. I was thinking of providing the flexibility that users pass these via xml

Reply:
Alright, I'll change it back and keep it as None by default.